sdk/formrecognizer/azure-ai-formrecognizer/azure/ai/formrecognizer/_form_recognizer_client.py

```diff
@@ -94,16 +101,16 @@ def begin_recognize_receipts(self, stream, **kwargs):
     @distributed_trace
     def begin_recognize_receipts_from_url(self, url, **kwargs):
         # type: (str, Any) -> LROPoller
-        """Extract field text and semantic values from a given receipt document.
-        The input document must be the location (Url) of the document to be analyzed.
-
+        """Extract field text and semantic values from a given US sales receipt.
+        The input document must be the location (Url) of the receipt to be analyzed.
 
         :param url: The url of the receipt. Currently only supports US sales receipts.
         :type url: str
         :keyword bool include_text_content: Include text lines and text content references in the result.
-        :return: LROPoller
+        :return: An instance of an LROPoller. Call `result()` on the poller
+            object to return the result of the long running operation.
         :rtype: ~azure.core.polling.LROPoller[list[~azure.ai.formrecognizer.USReceipt]]
-        :raises: ~azure.core.exceptions.HttpResponseError
+        :raises ~azure.core.exceptions.HttpResponseError:
```

Comment:
Why did you move the colon to the end? If you're adding it to the end why doesn't rtype have it at the end

Reply:
necessary for the reference to resolve. we do this in text analytic as well